<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>530</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>属性...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="apply_btn">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="norm_tab">
      <attribute name="title">
       <string>常规</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="14" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>游戏时间:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="nick_name_edit"/>
         </item>
         <item row="9" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="open_folder_btn">
             <property name="text">
              <string>打开文件夹</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="change_path_btn">
             <property name="text">
              <string>更改路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="change_icon_btn">
             <property name="text">
              <string>更改图标</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="10" column="0" colspan="2">
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLabel" name="last_run_label">
           <property name="text">
            <string>N\A</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>文件夹路径:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="game_name_edit"/>
         </item>
         <item row="15" column="0" colspan="2">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="icon_label">
           <property name="text">
            <string>无图标</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="game_name_label">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>game_name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="ps_edit"/>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="exe_path_edit"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>备注:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="12" column="0" colspan="2">
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>最后运行日期:</string>
           </property>
          </widget>
         </item>
         <item row="17" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="remove_game_btn">
             <property name="text">
              <string>移除游戏</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="game_folder_edit"/>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>游戏译名:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="change_bg_btn">
             <property name="text">
              <string>更改背景图片</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="edit_metadata_btn">
             <property name="text">
              <string>编辑游戏介绍</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="edit_txt_btn">
             <property name="text">
              <string>编辑攻略</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="14" column="1">
          <widget class="QLabel" name="total_time_label">
           <property name="text">
            <string>N\A</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>游戏路径:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>游戏名称:</string>
           </property>
          </widget>
         </item>
         <item row="16" column="0" colspan="2">
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ncd_tab">
      <attribute name="title">
       <string>存档云同步</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="1">
          <widget class="QLabel" name="sync_status_label">
           <property name="text">
            <string>N\A</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="save_date_label">
           <property name="text">
            <string>N\A</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="2">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>云同步状态：</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="Line" name="line_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>本地存档路径:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="10" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="open_savedata_btn">
             <property name="text">
              <string>打开存档目录</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="fix_ncd_btn">
             <property name="text">
              <string>修复云同步连接</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="use_ncd_checkbox">
           <property name="text">
            <string>启用存档云同步</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>云同步路径:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>云存档日期:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="sync_savepath_label">
           <property name="text">
            <string>N\A</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="savedata_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>N\A</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="2">
          <widget class="Line" name="line_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="Line" name="line_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="Line" name="line_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="adva_tab">
      <attribute name="title">
       <string>高级</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0" colspan="2">
          <widget class="Line" name="line_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="run_arg_edit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>运行参数:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
